### PR TITLE
Reformatted the "set_tmp_dir" in general.py

### LINKED ISSUE #6179: set_tmp_dir

### PR DESCRIPTION
**Pull request by AhmedSalah339**
Reformatted the "set_tmp_dir" in general.py
Close #6179

### CODE REVIEW

File: ivy/functional/ivy/general.py [hunk 1]
```diff
@@ -2088,13 +2088,25 @@ def get_tmp_dir():
...
+        The new directory for saving temporary files
+    Examples
+    --------
+    >>> x = ivy.get_tmp_dir()
+    >>> print(x)
+    /tmp
 
+    To set the temp base to 1e-04
+
+    >>> ivy.set_tmp_dir('/my_tmp')
```
Comment (hirwa-nshuti) [inline ivy/functional/ivy/general.py:2106]:
```suggestion
    >>> ivy.set_tmp_dir("/my_tmp")
```
Just putting strings in double quotes  as normal python string representation

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hi @AhmedSalah339 
First of thanks for contributing to Ivy and your PR is looking great😊

Just small changes and your PR gets merged.
Thanks🙂

**Comment by AhmedSalah339** [comment]
Thanks @hirwa-nshuti for the comment. 
Change added.

**Comment by hirwa-nshuti** [comment]
LGTM🙂

**Review by hirwa-nshuti — APPROVED** [review]

**Pull request merged**